content/en/blog/_posts/2023-07-18-swap-beta1-graduation/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: blog
+title: "Kubernetes 1.28: NodeSwap graduates to Beta"
+date: 2023-07-18
+slug: swap-beta1-1.28-2023
+---
+
+**Author:** Itamar Holder (Red Hat)
+
+The 1.22 release [introduced Alpha support](/blog/2021/08/09/run-nodes-with-swap-alpha/)
+for configuring swap memory usage for Kubernetes workloads on a per-node basis.
```

Comment:
Not quite right. Windows nodes have worked with swap space for some time now, and I think that's actually GA.

Reply:
Can you elaborate? IIUC swap just introduced in 1.22, before that kubelet would fail immediately if swap was spotted on the node. In addition, this sentence is actually taken from the [previous blog post](https://kubernetes.io/blog/2021/08/09/run-nodes-with-swap-alpha/#:~:text=The%201.22%20release%20introduced%20alpha%20support%20for%20configuring,for%20pod%20memory%20utilization%20when%20swap%20is%20involved.) for alpha support.

Am I missing anything?

Reply:
https://kubernetes.io/docs/concepts/configuration/windows-resource-management/#resource-management-memory says that Windows nodes use paging to disk.

Reply:
Wow, good to know! interesting!
I'll make it clear that we're talking specifically about Linux here.